tests/components/pyload/test_config_flow.py

```diff
@@ -0,0 +1,167 @@
+"""Test the pyLoad config flow."""
+
+from typing import Any
+from unittest.mock import AsyncMock
+
+from pyloadapi.exceptions import CannotConnect, InvalidAuth, ParserError
+import pytest
+
+from homeassistant.components.pyload.const import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_NAME
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN, HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+import homeassistant.helpers.issue_registry as ir
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.usefixtures("mock_pyloadapi")
```

Comment:
Just use it as parameter instead of annotation

Reply:
That's what edenhaus told me to do, because otherwise linter would complain about an unused variable

Reply:
I have never seen this though

Reply:
i mean also less imports for type hints are required, that a advantage

Reply:
Asyncmock is already imported